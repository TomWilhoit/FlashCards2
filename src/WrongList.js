import React, { Component } from "react";
import "./css/WrongList.css";

class WrongList extends Component {
  constructor(props) {
    super(props);
  }



  render() {
    console.log(this.props)
    const element = this.props.element;
    return (
      <ul>
        <li>{element.question}</li>
        <li>{element.correctAnswer}</li>
      </ul>
    );
  }
}

export default WrongList;
