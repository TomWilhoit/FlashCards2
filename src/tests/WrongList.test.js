import React from "react";
import WrongList from "../WrongList";
import { shallow } from "enzyme";

let element = {
  id: 3,
  question: "What is the difference between `const` and `let`?",
  correctAnswer: "With `const` the value cannot be reassigned",
  falseAnswer1: "With `let` the value cannot be reassigned",
  falseAnswer2: "Just `var` everything"
};

describe("WrongListCont", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<WrongList element={element} />);
  });
  it("should match snapshot when all data is passed correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
