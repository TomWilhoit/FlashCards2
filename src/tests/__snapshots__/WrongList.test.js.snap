// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WrongList should match snapshot when all data is passed correctly 1`] = `
<ul>
  <li>
    3
  </li>
  <li>
    What is the difference between \`const\` and \`let\`?
  </li>
  <li>
    With \`const\` the value cannot be reassigned
  </li>
</ul>
`;
